frontend/src/metabase/parameters/components/FilterApplyToast/FilterApplyToast.tsx

```diff
@@ -0,0 +1,73 @@
+import { useCallback } from "react";
+import { t } from "ttag";
+
+import { Toast } from "metabase/common/components/Toaster/Toaster";
+import { applyDraftParameterValues } from "metabase/dashboard/actions";
+import { setParameterValues } from "metabase/dashboard/actions/parameters";
+import { useDashboardContext } from "metabase/dashboard/context";
+import {
+  getDraftParameterValues,
+  getHasUnappliedParameterValues,
+  getIsAutoApplyFilters,
+  getParameterValues,
+} from "metabase/dashboard/selectors";
+import { useDispatch, useSelector } from "metabase/lib/redux";
+
+import { getFilterChangeDescription } from "./utils";
+
+interface FilterApplyToastProps {
+  position?: "absolute" | "fixed";
+}
+
+export function FilterApplyToast({
+  position = "absolute",
+}: FilterApplyToastProps) {
+  const isAutoApplyFilters = useSelector(getIsAutoApplyFilters);
+  const hasUnappliedParameterValues = useSelector(
+    getHasUnappliedParameterValues,
+  );
+  const currentParameterValues = useSelector(getParameterValues);
+  const draftParameterValues = useSelector(getDraftParameterValues);
+
+  const { isEditing } = useDashboardContext();
+
+  const dispatch = useDispatch();
+  const handleApplyFilters = useCallback(() => {
+    dispatch(applyDraftParameterValues());
+  }, [dispatch]);
+
+  const handleCancel = useCallback(() => {
+    dispatch(setParameterValues(currentParameterValues));
+  }, [currentParameterValues, dispatch]);
+
+  if (isAutoApplyFilters || !hasUnappliedParameterValues || isEditing) {
+    return null;
+  }
+
+  const filterChangeDescription = getFilterChangeDescription(
+    currentParameterValues,
+    draftParameterValues,
+  );
+
+  return (
+    <Toast
+      show
+      canClose={false}
+      data-testid="filter-apply-toast"
+      message={filterChangeDescription}
+      confirmText={t`Apply filter`}
```

Comment:
I wonder if we should just say "Apply" here. Cause if I'm removing an active filter "Apply filter" feels a bit funny.